core/crypto/SCsub

```diff
@@ -21,9 +21,9 @@ if is_builtin or not has_module:
 # to make a "light" build with only the necessary mbedtls files.
 if not has_module:
     # Minimal mbedTLS config file
-    env_crypto.Append(
-        CPPDEFINES=[("MBEDTLS_CONFIG_FILE", '\\"thirdparty/mbedtls/include/godot_core_mbedtls_config.h\\"')]
-    )
+    config_path = "thirdparty/mbedtls/include/godot_core_mbedtls_config.h"
+    config_path = f"<{config_path}>" if env_crypto["ninja"] and env_crypto.msvc else f'\\"{config_path}\\"'
+    env_crypto.Append(CPPDEFINES=[("MBEDTLS_CONFIG_FILE", config_path)])
```

Comment:
That's starting to be a bit convoluted...
I guess it's fine for now, but it would be worth figuring out why it's not being escaped properly by Ninja+MSVC, and how upstream mbedtls even expect this define to be passed in a way that won't break on some platforms.

Reply:
Yeah, it's not the prettiest solution & is very much a workaround. It should be reverted once the issue is ironed out in SCons, but that'll require some digging.